homeassistant/components/knx/__init__.py

```diff
@@ -191,20 +193,18 @@ def init_xknx(self):
         """Initialize of KNX object."""
         self.xknx = XKNX(
             config=self.config_file(),
-            loop=self.hass.loop,
```

Comment:
Python 3.6 support has been dropped. We now use `asyncio.get_running_event_loop()`.